### PR TITLE
chore(SM-127): add project scaffold

### PR DESCRIPTION
**Pull request by Polpetta**
* Add Jenkins integration
* Add pre-commit integration
* Create first scaffold

I open the PR as a draft while I add last things

### CODE REVIEW

**Review by keshavbhatt — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  </build>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>testcontainers</artifactId>
+      <version>${org.testcontainers.junit-jupiter.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.testcontainers</groupId>
```
Comment (frisonisland) [inline pom.xml:107]: Test scope?

Comment (Polpetta) [reply pom.xml:107]: You're very right :)

Comment (Polpetta) [reply pom.xml:107]: Done!

**Comment by Polpetta** [comment]
Note: release drafter fails because its definition is not in the `main` branch yet...so it will actually fail always for this PR :)

**Review by aheeva-yuliya — APPROVED** [review]

**Pull request merged**